src/OpenConext/ProfileBundle/Resources/views/MyConnections/overview.html.twig

```diff
@@ -24,11 +24,11 @@
                 <div class="mdl-grid">
                     <div class="mdl-cell mdl-cell--3-col">
                         <div class="mdl-layout-title">{{ 'profile.my_connections.service_label'|trans }}</div>
-                        <img src="{{ connection.logoPath }}" title="{{ 'profile.my_connections.orcid.title'|trans }}"/>
+                        <img src="{{ connection.logoPath }}" class="logo" title="{{ 'profile.my_connections.orcid.title'|trans }}"/>
```

Comment:
This may be out of the scope for this PR, but these images don't have an alt text. Adding them may benefit users who use a screen reader, and make it clear what *should* be there if the image doesn't load

Reply:
Yes, a good suggestion. Fixed that for the connect/disconnect logo.